frontend/src/v5/ui/routes/dashboard/projects/drawings/uploadDrawingRevisionForm/uploadList/uploadListItem/components/uploadListItemStatusCode/uploadListItemStatusCode.component.tsx

```diff
@@ -63,7 +63,6 @@ export const UploadListItemStatusCode = ({ value, inputRef, onChange, helperText
 				/>
 			)}
 			ListboxComponent={OptionsBox}
-			disableClearable={!value}
```

Comment:
When the preset drawing gets cleared, the "clearable" button persists


https://github.com/user-attachments/assets/07554506-30c1-4702-9429-7cbde4a01e9b